src/pocketmine/level/Level.php

```diff
@@ -1230,12 +1230,9 @@ public function rayTraceBlocks(Vector3 $pos1, Vector3 $pos2, $flag = false, $fla
 	public function getFullLight(Vector3 $pos) : int{
 		$chunk = $this->getChunk($pos->x >> 4, $pos->z >> 4, false);
 		$level = 0;
-		if($chunk !== null){
-			$level = $chunk->getBlockSkyLight($pos->x & 0x0f, $pos->y & 0x7f, $pos->z & 0x0f);
-			//TODO: decrease light level by time of day
-			if($level < 15){
-				$level = max($chunk->getBlockLight($pos->x & 0x0f, $pos->y & 0x7f, $pos->z & 0x0f));
-			}
+		if($chunk instanceof FullChunk){
+			//Do not decrease skylightlvl over day. MCPC and MCPE don't do this either.
```

Comment:
I don't understand the blah about lighting curves because my brain isn't properly awake, but http://minecraft.gamepedia.com/Light#Brightness seems like a must-read.